lib/start_video_demos.exs

```diff
@@ -0,0 +1,25 @@
+first_raw_video_path = "~/membrane_video_compositor/testsrc.raw"
+second_raw_video_path = "~/membrane_video_compositor/testsrc.raw"
+output_path = "~/membrane_video_compositor/output.h264"
+
+video_width = 1280
+video_height = 720
+video_framerate = 30
+# implementation is one of: :ffmpeg, :opengl, :nx
```

Comment:
Should be obvious from the `VideoCompositor` docs / options - I think this comment is unecessary